### DISCUSSION TITLE
How do you stop propagation on RNGH's Touchable's?

### DISCUSSION THREAD

**Discussion by jamesehly** [software-mansion/react-native-gesture-handler#1972] [Q&A]
I have a RNGH TouchableOpacity nested within another RNGH TouchableOpacity.  When clicking on the inner TouchableOpacity the onPress event fires for both.  How can I stop propagation from the child to parent? RNGH Touchables don't seem to pass an event with the stopPropagation function on it, so, is this not possible at the moment? Does anyone have some insight on how this can be achieved, or is the solutionto not nest RNGH Touchables?

**Comment by fmorau** [comment]
> I have a RNGH TouchableOpacity nested within another RNGH TouchableOpacity. When clicking on the inner TouchableOpacity the onPress event fires for both. How can I stop propagation from the child to parent? RNGH Touchables don't seem to pass an event with the stopPropagation function on it, so, is this not possible at the moment? Does anyone have some insight on how this can be achieved, or is the solutionto not nest RNGH Touchables?

I faced the same issue, and looking for how to solve it.
Did you find a solution?
